### PR TITLE
Allow multiple robots to be controlled by independent action servers.

### PR DESCRIPTION
**Pull request by tigressine**
Solves #371

### CODE REVIEW

File: packages/extras/ezrassor_launcher/launch/configurable_simulation.launch [hunk 1]
```diff
@@ -120,6 +117,5 @@
     <arg name="use_sim_time" value="$(arg use_sim_time)"/>
     <arg name="enable_real_odometry" value="$(arg enable_real_odometry)"/>
     <arg name="enable_park_ranger" value="$(arg enable_park_ranger)"/>
-    <arg name="world" value="$(arg world)"/>
```
Comment (tigressine) [inline packages/extras/ezrassor_launcher/launch/configurable_simulation.launch:123]: This line was a bug, it already exists a couple of lines up. Prevented multiple robots from spawning (errored out).

File: packages/actions/ezrassor_teleop_actions/source/ezrassor_teleop_actions/teleop_action_server.py [hunk 2]
```diff
@@ -16,14 +16,15 @@
...
+QUEUE_SIZE = 10
 
 
 class TeleopActionServer:
     def __init__(self):
 
         # ROS Actionlib Library
         self._server = actionlib.SimpleActionServer(
-            "/teleop_action_server",
+            "teleop_action_server",
```
Comment (tigressine) [inline packages/actions/ezrassor_teleop_actions/source/ezrassor_teleop_actions/teleop_action_server.py:27]: removed the forward slash to allow the server topics to be namespaced.

File: packages/actions/ezrassor_teleop_actions/source/ezrassor_teleop_actions/teleop_action_server.py [hunk 3]
```diff
@@ -64,6 +77,11 @@ def __init__(self):
         # Init the timer counter
         self._counter = 0
 
+        # Initialize positional/state data.
```
Comment (tigressine) [inline packages/actions/ezrassor_teleop_actions/source/ezrassor_teleop_actions/teleop_action_server.py:80]: By initializing here, we can run the action server without the sim. The feedback won't be accurate of course, but it allows us to test it without having to launch the sim (by echoing topics etc).

**Comment by tigressine** [comment]
Documentation updates:
- launch files changes (new params)
- update inputs/outputs for new namespaced topics
- update conf_sim and conf_comm with changes to inputs
- create issue to find a program or service that visualizes a complex graph, so we could have an interactive way to view the architecture
- info about namespacing client

**Review by CSharpRon — APPROVED** [review]
Great work on these changes, the only places I had questions, you cleared them up with your review comments.

**Pull request merged**